integrations/google/creds.go

```diff
@@ -123,6 +123,11 @@ func (h handler) finalize(ctx context.Context, c sdkintegrations.ConnectionInit,
 		return
 	}
 
+	// Reset the list of vars because "h.vars.Set()" modifies the values of secret vars.
```

Comment:
wait what, Set modified the inputs?!

Reply:
this doesn't make sense as all sdktypes are immutable

Reply:
https://github.com/autokitteh/autokitteh/blob/main/internal/backend/vars/vars.go#L43

Reply:
oh got it.